### PR TITLE
docs: order csp strictly alphabetically

### PR DESCRIPTION
**Pull request by m1ghtym0**
### Context

Order CSPs alphabetically and replace discord link with gh discussions.

### Checklist

- [x] Run the E2E tests that are relevant to this PR's changes
- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [x] Add labels (e.g., for changelog category)

### CODE REVIEW

**Comment by m1ghtym0** [comment]
@thomasten @malt3 Should we rename GCP into Google Cloud? While GCP is still commonly used it seems not to be the official term anymore.

File: docs/docs/workflows/scale.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ kubectl -n kube-system get nodes
 Alternatively, you can manually scale your cluster up or down:
```
[the remark this answers is not in the document]
Comment (malt3) [reply docs/docs/workflows/scale.md:52]: We could write that is is currently not supported or coming in the future.

**Comment by malt3** [comment]
> @thomasten @malt3 Should we rename GCP into Google Cloud? While GCP is still commonly used it seems not to be the official term anymore.

I'm fine with doing that in the docs. We should keep constants used in the code.

**Comment by thomasten** [comment]
> > @thomasten @malt3 Should we rename GCP into Google Cloud? While GCP is still commonly used it seems not to be the official term anymore.
> 
> I'm fine with doing that in the docs. We should keep constants used in the code.

Exactly :)

**Review by thomasten — APPROVED** [review]

**Pull request merged**